c_src/ex_libsrtp/unifex_util.c

```diff
@@ -12,3 +13,8 @@ UnifexPayload *unifex_payload_clone_ex(UnifexEnv *env, UnifexPayload *original,
 UnifexPayload *unifex_payload_clone(UnifexEnv *env, UnifexPayload *original) {
   return unifex_payload_clone_ex(env, original, original->type, original->size);
 }
+
+void unifex_payload_free_clone(UnifexPayload *payload) {
```

Comment:
Not sure if this function should be calle `unifex_payload_free_clone` or `unifex_payload_clone_free`.